code/game/objects/items/weapons/misc.dm

```diff
@@ -135,11 +135,11 @@
 	throw_speed = 1
 	throw_range = 4
 	w_class = 2
-	attack_verb = list("called", "rang")
+	attack_verb = list("called", "rang", "dialed", "belled")
```

Comment:
Don't really like these, to be honest--it kinda takes away from the "there's a call, it's for you" *caves head in with a telephone* joke. :stuck_out_tongue: